.github/workflows/security_owasp.yml

```diff
@@ -38,49 +38,22 @@ jobs:
       id: owasp-analyse
       run: ./gradlew ${{ inputs.subproject }}:dependencyCheckAnalyze --info
       continue-on-error: true
-
-    - uses: ministryofjustice/hmpps-github-actions/.github/actions/security_owasp_reports@v0.4 # WORKFLOW_VERSION
-      if: steps.owasp-analyse.outcome == 'failure'
+    - uses: github/codeql-action/upload-sarif@v3
```

Comment:
move the separate owasp reports action here as only used by this workflow anyway